org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/SwitchPatternTest21.java

```diff
@@ -2,19 +2,12 @@
 
 import java.util.Map;
 
-import org.eclipse.jdt.internal.compiler.classfmt.ClassFileConstants;
 import org.eclipse.jdt.internal.compiler.impl.CompilerOptions;
 
 import junit.framework.Test;
 
 public class SwitchPatternTest21 extends AbstractBatchCompilerTest {
 
-	private static String[] JAVAC_OPTIONS = new String[] { "--enable-preview" };
-
-	static {
-		//	TESTS_NAMES = new String [] { "testNaming" };
-	}
```

Comment:
this block helps running individual tests, so we may want to keep it :)

Reply:
Restored.